### ISSUE TITLE
Color Gamut option missing after days of troubleshooting.

### ISSUE THREAD

**Issue by Gidrahwastaken** [seerge/g-helper#3461]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

The color gamut option is missing in G Helper and even Asus Armoury Crate on my Asus ROG Strix Scar 17 2023 G733PY laptop. I really hope I can get help here because this is driving me crazy because my screen is DCI-P3 and I want to use sRGB since everything looks oversaturated.

Here is a list of everything I have tried but no luck.
1. Clean install of Windows 11 Pro 3 separate times, once local, once cloud, and once not installing armoury crate. Including trying windows versions 23H2 and 24H2.
2. Clean install of G Helper without armoury crate or myasus every being installed on the system.
3. Tried messing with color management and installing color profiles from the Game Visual folder but no luck.
4. Made sure all drivers are up to date.
5. Checking MyAsus drivers (Color Gamut options and Splendid does not show up here either) 
6. Checked to make sure Asus Splendid is installed.
7. I tried the solution [here](https://www.asus.com/us/support/faq/1050541/) but I already have the ICM files and asus support just told me to try an EC reset which did not work.

The only thing I haven't tried is installing older versions of G Helper to see if this is just a current version issue which I doubt since amoury crate is having the same issue. Only other thing I could think of is that since this is the AMD version of the laptop I may not have access to this feature? I am also missing Auto Color Management in windows Advanced Display settings and read that was related to it not working with AMD processors. 

Side note: I will also say that my laptop is having severe overheating issues unless I limit the voltage that 55W and undervolt and even then it idles around 50-70c but I think the liquid metal may need to be reapplied. 

Any help would be appreciated, or if anyone else also shows color gamut missing it would be incredibly validating if I am not the only one with this issue and its widespread.

### How to reproduce the bug?

The problem persists not matter what I do.

### Logs

[log.txt](https://github.com/user-attachments/files/18082141/log.txt)

### Device and Model

Asus Rog Strix Scar 17 (2023) G733PY

### Additional information.

<img width="1280" alt="gamevisual no gamut" src="https://github.com/user-attachments/assets/6f450f7d-835b-4ee2-850a-df2d51cd6c89">

### Armoury Crate

Uninstalled

### Asus Services

Tried both default and 0 services running

### Version

v0.199

### OS

Windows 11 Pro 24H2

**Comment by Gidrahwastaken** [comment]
![Uploading gamevisual no gamut option g helper.png…]()

**Comment by Gidrahwastaken** [comment]
<img width="1280" alt="gamevisual no gamut option g helper" src="https://github.com/user-attachments/assets/1925300b-92bf-4c17-86c7-63ada7858fb9">

**Comment by seerge** [comment]
@Gidrahwastaken hello,

Did you ever had this options in AC or G-Helper ? Cause many models simply don't have sRGB or other gamuts.

G-Helper just looks at available color profiles in `C:\ProgramData\ASUS\GameVisual` (where asus stores them). And if they exist - it lists options. Version of the app won't make any difference of course

**Comment by Gidrahwastaken** [comment]
> @Gidrahwastaken hello,
> 
> Did you ever had this options in AC or G-Helper ? Cause many models simply don't have sRGB or other gamuts.
> 
> G-Helper just looks at available color profiles in `C:\ProgramData\ASUS\GameVisual` (where asus stores them). And if they exist - it lists options. Version of the app won't make any difference of course

I have never had it since I got the laptop a week ago. I do know for sure that my my screen is DCI-P3 according to laptop specs and because colors are oversaturated. I can confirm this because novideo_srgb will not clamp the gamut to the srgb color space if its already on it. 

It seems very odd that they would give this model a wide gamut and not give the option to use the color space 90% of applications are made for.

**Comment by Gidrahwastaken** [comment]
Yeah no luck but thanks, I've pretty much given up on that issue since novideo_srgb clamps the gamut fine. 

My real problem is that brightness is about 50 nits brighter when using the DGPU vs the iGPU making whites too bright and black level/shadow detail too dark. I could run Dynamic in bios for browsing bit that doesn't solve the issue in games, not being able to spot enemies in dark areas and flashbangs being brighter is not fun.

Currently downgrading a fresh copy of W11 23H3 to see if its just a 24h2 bug but its extremely weird behavior considering my old HP Omen doesn't change brightness at all when switching between the iGPU and DGPU. 

This is my 2nd asus laptop and I swear ill never buy this brand again, has been nothing but a headache.

**Comment by Gidrahwastaken** [comment]
Hopefully this can help someone for the screen brightness issue. It seems setting GPU Only in the bios enables some kind of pseudo HDR brightness which is odd because this display cant do HDR. That explains why whites were super bright. Using standard and setting DGPU only in Nvidia settings returns normal brightness levels while maintaining performance.

Also note to seerge because I don't want to open a new thread. Im not sure if this from G Helper or something else but in battery options my minimum processor state while plugged in was set to 80% instead of the default 5% causing high temps while idle. This behavior only started after G Helper was installed and of intentional id highly encourage setting that value back to 5% so the CPU can downclock when not in use. 

All this took a week of troubleshooting, I hope it helps someone.

**Comment by seerge** [comment]
@Gidrahwastaken hello,

> GPU Only in the bios

Is just an Ultimate / Standard toggle (but in BIOS). It routes internal display to dGPU. What you observe sounds like a driver issue tho that pops up only in certain situations

> Also note to seerge because I don't want to open a new thread. Im not sure if this from G Helper or something else but in battery options my minimum processor state while plugged in was set to 80% instead of the default 5% causing high temps while idle.

Processor state is a setting in windows Power Plans. If you have modified it before while using AC, most likely you edited other power plan. And now with G-Helper system uses default "Balanced" power plan, that you didn't edit yet. So you see 80% there.

G-Helper doesn't have any "default" setting for power plan. It just shows current state of the things, like what kind of Boost setting do you already have selected.

> P.S. Is 300/1500 the max GPU overclock limit or can it go higher?

You [can go higher ](https://github.com/seerge/g-helper/wiki/Power-user-settings#higher-maximum-and-minimum-gpu-clock--memory-offsets)

> Thank you for all your work on this app, I donated. <3

Thanks :)

**Comment by Solarriors** [comment]
Just to add more honest information. This happened to me just now. But it's me who changed/messed with color profiles through G-Helper, Windows Settings, Color Management Panel at the same time, swapping profiles in each of the window simulatneously. I added and removed profiles on the CMP continously, and swapped one profile in one instance and changed to another in another instance. E.g. I select Native in G-Helper, then select sRGB in Windows Settings, then make DCI-P3 default in CMP, then add another sRGB standard in CMP, then select sRGB in G-Helper, and many different combinations until some of the profiles disappeared, got overwritten, corrupt or conflicted.
I just did as above, delete Game Visual files and restart G-Helper to download them. I don't know if there's any necessity to do through a redownload of the ASUS Nvidia Drivers as I don't use them since they are a bit obsolete in comparison to the current official Nvidia.
